Gemfile

```diff
@@ -111,3 +111,24 @@ gem "logstash-output-stdout"
 gem "logstash-output-tcp"
 gem "logstash-output-udp"
 gem "logstash-output-webhdfs"
+gem "logstash-filter-multiline"
+gem "jar-dependencies", ">= 0"
+gem "ruby-maven", "~> 3.3"
+gem "logstash-patterns-core", ">= 0"
+gem "pry", ">= 0"
+gem "rspec-wait", ">= 0"
+gem "childprocess", ">= 0"
+gem "ftw", "~> 0.0.42"
+gem "rspec-sequencing", ">= 0"
+gem "gmetric", ">= 0"
+gem "gelf", "= 1.3.2"
+gem "timecop", ">= 0"
+gem "jdbc-derby", ">= 0"
+gem "docker-api", ">= 0"
+gem "longshoreman", ">= 0"
+gem "elasticsearch", ">= 0"
+gem "sinatra", ">= 0"
+gem "webrick", ">= 0"
+gem "poseidon", ">= 0"
+gem "snappy", ">= 0"
+gem "webmock", "~> 1.21.0"
```

Comment:
this is probably left from installing something, let make sure we dont merge it in master.